ta/pkcs11/src/entry.c

```diff
@@ -136,4 +149,12 @@ TEE_Result TA_InvokeCommandEntryPoint(void *tee_session __unused, uint32_t cmd,
 		ctrl->memref.size = 0;
 
 	return res;
+
+bad_types:
+	DMSG("Bad parameter types used at PKCS11 TA entry:");
```

Comment:
I'm not sure this "usage" helps very much. The only expected caller is from within the libcktee and that caller is supposed to know this stuff already.

Reply:
true. will discard the change.